node/src/blockchain/blockchain_interface.rs

```diff
@@ -1689,31 +1711,35 @@ mod tests {
     }
 
     #[test]
-    fn non_clandestine_gas_limit_for_polygon_mainnet_lies_within_limits() {
+    fn non_clandestine_gas_limit_for_polygon_mainnet_lies_within_limits_for_raw_transaction() {
```

Comment:
This test and the next one could be mostly combined...although for just two, it's not as urgent as it would be for forty-seven.